src/HttpCache/CsvFormatter.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\HttpCache;
+
+/**
+ * Formats the cache header as a comma-separated list.
+ *
+ * @author Yanick Witschi <yanick.witschi@terminal42.ch>
+ */
+class CsvFormatter implements CacheTagsFormatterInterface
```

Comment:
I'm think it is not worth it to add a new class for an implode (see my other suggestion).